hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/RequestFactoryImpl.java

```diff
@@ -584,14 +586,17 @@ public ListObjectsV2Request newListObjectsV2Request(
       final String key,
       final String delimiter,
       final int maxKeys) {
-    final ListObjectsV2Request request = new ListObjectsV2Request()
-        .withBucketName(bucket)
-        .withMaxKeys(maxKeys)
-        .withPrefix(key);
+
+    final ListObjectsV2Request.Builder requestBuilder =
+        ListObjectsV2Request.builder().bucket(bucket).maxKeys(maxKeys).prefix(key);
```

Comment:
I'd recommend in general to keep each builder arg on a new line similar to how it was previously.

```suggestion
    final ListObjectsV2Request.Builder requestBuilder = ListObjectsV2Request.builder()
        .bucket(bucket)
        .maxKeys(maxKeys)
        .prefix(key);
```